app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt

```diff
@@ -98,7 +98,7 @@ class RecentlyPlayedFragmentPresenter @Inject constructor(
         )
         else -> {
           logger.d("RecentlyPlayedFragment", "Successfully loaded exploration")
-          routeToExplorationListener.routeToExploration(internalProfileId, topicId, storyId, explorationId)
+          routeToExplorationListener.routeToExploration(internalProfileId, topicId, storyId, explorationId, /* backflowScreen = */null)
```

Comment:
Nit: Spacing issue there should be space before `null`
`/* backflowScreen = */ null`

Reply:
Done